### ISSUE TITLE
[Bug][A11Y][Password input] Show/hide button not reachable with external keyboard

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#854]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone

### OS Version

26

### App version

2.0.0 (708)

### Describe the issue

on iOS only, the "show/hide password" button is not focused anymore with Full Keyboard Access
No problem with Voice Over

### Expected Behavior

"Show/hide password" button shall be focused just after the associated input
